docs/ecology.md

```diff
@@ -1 +1,176 @@
-# Victory
\ No newline at end of file
+# Victory
+
+Victory is a collection of composable React components you can use to build interactive data visualizations.
+
+### Overarching benefits and motivations
+
+1. Bridge the gap between beginners (more likely to use Chart.js or Highcharts) and experts (more likely to use D3)
+  a. Provide a gradient of customization, from none (just pass in data) to full (build your own visualization features)
+  b. Provide interactive documentation to help developers get visualizations into their projects (see [ecology.js](https://github.com/FormidableLabs/ecology)) see [victory-chart](projects.formidablelabs.com/victory-chart/) for an example
+  c. Provide better developer UX generally, for both instantiating data visualizations and creating them
+2. Leverage the power of React to handle DOM, component lifecycle, state, computed styles, etc.
+3. Make data visualizations first class citizens in the open source community
+  a. Repos rather than gists
+  b. forking / issues for continual improvement
+  c. `package.json` for dependencies
+  d. minified dists / umd builds for CDN
+  e. Babel for ES6
+  f. Webpack for builds
+  g. `propTypes` for input validation
+  h. Hot Reloading
+  i. sourcemaps
+  j. `eslintrc`
+  k. Tests
+
+### Using Victory
+
+Victory components can be dropped into your project in seconds. Here's an example, using `victory-pie`.
+
+In your terminal:
+
+`$ npm install victory-pie`
+
+In a view in your React app...
+
+`import {VictoryPie} from "victory-pie";
+
+...
+
+```
+render() {
+  <VictoryPie/>
+}
+
+```
+
+...will render a victory component with default data. The data visualization can then be configured with props...
+
+```
+<VictoryPie
```

Comment:
I like leaving an interactive example here, and we should show how to include all of victory and pull parts out of it like in your workshop example